app/main.py

```diff
@@ -1 +1,47 @@
-#  write your code here
+class Animal:
+
+    def __init__(
+            self, name: str, appetite: int, is_hungry: bool = True
+    ) -> None:
+        self.name = name
+        self.appetite = appetite
+        self.is_hungry = is_hungry
+
+    def print_name(self) -> None:
+        print(f"Hello, I'm {self.name}")
+
+    def feed(self) -> int:
+        if self.is_hungry:
+            print(f"Eating {self.appetite} food points...")
+            self.is_hungry = False
+            return self.appetite
+        return 0
```

Comment:
I mean... its ok, but what if your logic was 100 lines of code?
In such scenarios you can think in reverse, f.e.
`if not self.is_hungry` 
and after main logic